packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,223 @@
+
+Each navigation item provided must be associated to a path that is present within the `routingConfig` specification. The navigation item, as rendered by the ApplicationLayout, will route to that path upon selection.
+
+The value provided for `navigationItems` should be an array of objects with the following API:
+
+|Key Name|Type|Is Required|Description|
+|---|---|---|---|
+|`text`|String|required|A string rendered within the navigation item control.|
+|`path`|Element|required|A string path to route to upon navigation item selection. This path should be present within the `routingConfig` as well.|
+
+```jsx
+/**
+ * Given the above routingConfig example, with entries defined for paths '/page_1' and '/page_2',
+ * the following configuration would generate controls for moving between those two paths.
+ */
+const navigationItems = [{
+  path: '/page_1',
+  text: 'Page 1',
+}, {
+  path: '/page_2',
+  text: 'Page 2',
+}];
+```
+
+### `extensions`
+
+The `extensions` prop allows consumers to render an element within the ApplicationLayout's extensions region. The `extensions` content will be rendered in various ways depending on the current breakpoint. The `extensions` element will receive `app` and `layoutConfig` props from the ApplicationLayout, allowing it to interact with and respond to changes within the ApplicationLayout.
+
+```jsx
+import { Utils } from 'terra-application-layout';
+
+const MyExtensions = ({ app, layoutConfig }) => {
+  if (Utils.helpers.isSizeCompact(layoutConfig.size)) {
+    return (
+      <div>Extensions for ApplicationLayout menu!</div>
+    );
+  } else {
+    return (
+      <div>Extensions for ApplicationLayout header!</div>
+    );
+  }
+};
+```
+
+### `indexPath`
+
+The `indexPath` prop allows consumers to set the default path of the ApplicationLayout. The ApplicationLayout will redirect to this path should users of the consuming application attempt to route to a component not detailed in the `routingConfig` prop. Accordingly, the `indexPath` value must have an associated entry within the `routingConfig` specification.
+
+### `app`
+
+By default, the ApplicationLayout will render itself within a `ModalManager` and provide modal presentation capabilities to all of its child components. However, the ApplicationLayout accepts an AppDelegate prop (as `app`) that it will fall back to, if necessary. If no `app` prop is provided, the ModalManager provided by the ApplicationLayout will handle all disclosure requests by the ApplicationLayout's children.
+
+## Responsive Design
+
+The ApplicationLayout has two rendering modes: `standard` and `compact`.
+  - The `standard` rendering occurs at `medium`, `large`, and `huge` breakpoints.
+  - The `compact` rendering occurs at `tiny` and `small` breakpoints.
+
+Depending on the rendering mode, the ApplicationLayout's prop data is rendered in different ways.
```

Comment:
Do you need this statement anymore? It feels like an incomplete thought, but I think your list above describes this.

Reply:
I'll axe this and roll directly into the table.